client/src/locales/en-US.json

```
@@ -658,7 +658,7 @@
             },
             "configuration": {
                 "outsidersAllowed": "External profile",
-                "userLimit": "User limit",
+                "userLimit": "User limit (excluding external profiles)",
```

Profile should be singular and External capitalized
```suggestion
                "userLimit": "User limit (excluding External profile)",
```
Or else
```suggestion
                "userLimit": "User limit (excluding users with External profile)",
```